docs/source/release/v6.7.0/framework.rst

```diff
@@ -49,20 +49,20 @@ New features
 
 Bugfixes
 ########
-- Added throw statement in case of index overflow in isMonitor methods
+- Added throw statement in case of index overflow in ``isMonitor`` methods to avoid unintentional behaviour.
```

Comment:
This seems like an implementation detail. Maybe this would be better?
```suggestion
- ``isMonitor`` methods will now produce an error if an index overflow occurs, to avoid unintentional behaviour.
```